.travis.yml

```diff
@@ -4,7 +4,11 @@ python:
   - "2.7"
 
 install:
+    - pip install flake8
     - pip install -r requirements.txt
```

Comment:
This works, but better add flake8 to requirements.txt and Pipefile's dev-packages section, so we can run flake8 locally.